psibomana/react-redux-project
# Component tests

**What does this PR do?**
Add test cases to the application

**Description of Task to be completed?**
Add test cases for the application

**How should this be manually tested?**
Clone and set up the project using the instruction in the README.
Use `$ git checkout component_tests` to check out the branch containing the changes of this pull request
Use `$ yarn test` to run the application tests
All the tests should run successfully.
Use `$ yarn test:coverage` to run the application tests and generate code coverage report
